src/d/actor/d_a_player_dproc.inc

```diff
@@ -205,483 +507,1867 @@ BOOL daPy_lk_c::setGetDemo() {
+      }
+      return true;
 }
 
 /* 80130A54-80130ADC       .text dProcNod_init__9daPy_lk_cFv */
 BOOL daPy_lk_c::dProcNod_init() {
-    /* Nonmatching */
+    if (mCurProc == daPyProc_DEMO_NOD_e) {
+        return true;
+      } else {
+        commonProcInit(daPyProc_DEMO_NOD_e);
+        mVelocity = 0.0f;
+        setSingleMoveAnime(ANM_SEYYES, daPy_HIO_turn_c0::m.field_0x1C,
+                           daPy_HIO_turn_c0::m.field_0x20,
+                           daPy_HIO_turn_c0::m.field_0xC,
+                           daPy_HIO_turn_c0::m.field_0x24);
+        dComIfGp_setPlayerStatus0(0, daPyStts0_UNK10_e);
+      }
+      return true;
 }
 
 /* 80130ADC-80130B38       .text dProcNod__9daPy_lk_cFv */
 BOOL daPy_lk_c::dProcNod() {
-    /* Nonmatching */
+    setShapeAngleToTalkActor();
+    current.angle.y = shape_angle.y;
+    if (mFrameCtrlUnder[UNDER_MOVE0_e].getRate() < 0.0099999998f) {
+      dComIfGp_evmng_cutEnd(mStaffIdx);
+    }
+    return true;
 }
 
 /* 80130B38-80130BE8       .text dProcPresent_init_sub__9daPy_lk_cFv */
 void daPy_lk_c::dProcPresent_init_sub() {
-    /* Nonmatching */
+    setSingleMoveAnime(ANM_PRESENTATIONA, daPy_HIO_turn_c0::m.field_0x28,
+                       daPy_HIO_turn_c0::m.field_0x2C,
+                       daPy_HIO_turn_c0::m.field_0xE,
+                       daPy_HIO_turn_c0::m.field_0x30);
+    onModeFlg(ModeFlg_00000400);
+    if (checkBottleItem(dComIfGp_event_getPreItemNo()) == 0) {
+      dComIfGp_event_setItemPartnerId(fopAcM_createItemForPresentDemo(
+          &current.pos, dComIfGp_event_getPreItemNo(), 3, -1,
+          fopAcM_GetRoomNo(this), &shape_angle, &scale));
+    }
+    keepItemData();
+    return;
 }
 
 /* 80130BE8-80130CA0       .text dProcPresent_init__9daPy_lk_cFv */
 BOOL daPy_lk_c::dProcPresent_init() {
-    /* Nonmatching */
+    if (mCurProc == daPyProc_DEMO_PRESENT_e) {
+      return true;
+    } else {
+      commonProcInit(daPyProc_DEMO_PRESENT_e);
+      m3570 = setTalkStartBack();
+      if (m3570 == 0) {
+        mVelocity = 0.0f;
+        dProcPresent_init_sub();
+      }
+      dComIfGp_setPlayerStatus0(0, daPyStts0_UNK10_e);
+      mDoAud_seStart(JA_SE_TALK_START);
+      m3574 = 0;
+    }
+    return true;
 }
 
 /* 80130CA0-80130E88       .text dProcPresent__9daPy_lk_cFv */
 BOOL daPy_lk_c::dProcPresent() {
-    /* Nonmatching */
+    setShapeAngleToTalkActor();
+    if (m3570 != 0) {
+      cXyz local_28 = (m370C - current.pos);
+      if ((0x4000 < cLib_distanceAngleS(cM_atan2s(local_28.x, local_28.z),
+                                        current.angle.y)) ||
+          mAcch.ChkWallHit()) {
+        current.pos.x = m370C.x;
+        current.pos.z = m370C.z;
+        m3570 = 0;
+        dProcPresent_init_sub();
+        mVelocity = 0.0f;
+      }
+    } else {
+      current.angle.y = shape_angle.y;
+      if (m3574 != 0) {
+        dComIfGp_evmng_cutEnd(mStaffIdx);
+      } else {
+        if (mFrameCtrlUnder[UNDER_MOVE0_e].getRate() < 0.0099999998f) {
```

Comment:
Some of these `else { if`s can be condensed into `else if`s like this
```suggestion
      } else if (mFrameCtrlUnder[UNDER_MOVE0_e].getRate() < 0.01f) {
```
That should reduce the indentation levels in this function a bit